discourse_rock/patches/discourse-charm.patch

```diff
@@ -0,0 +1,18 @@
+--- a/lib/tasks/discourse-charm.rake
++++ b/lib/tasks/discourse-charm.rake
+@@ -0,0 +1,15 @@
++# frozen_string_literal: true
++
++desc "Check if a user exists for given email address"
++task "users:exists", [:email] => [:environment] do |_, args|
++  email = args[:email]
++  if !email || email !~ /@/
++    puts "ERROR: Expecting rake users:exists[some@email.com]"
++    exit 1
++  end
++  if User.find_by_email(email)
++    puts "ERROR: User with email #{email} not found"
```

Comment:
This line does the opposite of what is expected, the previous condition is if the user exists and then it states that the user does not exists.